[2025-09-21T23:54:33.046+0000] {processor.py:161} INFO - Started process (PID=67) to work on /opt/airflow/dags/processamento_silver.py
[2025-09-21T23:54:33.048+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/processamento_silver.py for tasks to queue
[2025-09-21T23:54:33.051+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:54:33.051+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/processamento_silver.py
[2025-09-21T23:54:33.091+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:54:33.086+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/customers_to_silver.sql
[2025-09-21T23:54:33.092+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:54:33.092+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/geolocation_to_silver.sql
[2025-09-21T23:54:33.092+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:54:33.092+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/order_items_to_silver.sql
[2025-09-21T23:54:33.093+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:54:33.093+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/order_payments_to_silver.sql
[2025-09-21T23:54:33.094+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:54:33.093+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/order_reviews_to_silver.sql
[2025-09-21T23:54:33.094+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:54:33.094+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/orders_to_silver.sql
[2025-09-21T23:54:33.095+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:54:33.095+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/products_to_silver.sql
[2025-09-21T23:54:33.100+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:54:33.099+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/sellers_to_silver.sql
[2025-09-21T23:54:33.100+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:54:33.100+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/product_category_name_translation_to_silver.sql
[2025-09-21T23:54:33.141+0000] {processor.py:840} INFO - DAG(s) 'processamento_silver' retrieved from /opt/airflow/dags/processamento_silver.py
[2025-09-21T23:54:33.584+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:54:33.583+0000] {override.py:1769} INFO - Created Permission View: can edit on DAG:processamento_silver
[2025-09-21T23:54:33.601+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:54:33.601+0000] {override.py:1769} INFO - Created Permission View: can delete on DAG:processamento_silver
[2025-09-21T23:54:33.612+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:54:33.612+0000] {override.py:1769} INFO - Created Permission View: can read on DAG:processamento_silver
[2025-09-21T23:54:33.613+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:54:33.613+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-09-21T23:54:33.632+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:54:33.631+0000] {dag.py:3058} INFO - Creating ORM DAG for processamento_silver
[2025-09-21T23:54:33.634+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:54:33.634+0000] {dag.py:3823} INFO - Setting next_dagrun for processamento_silver to None, run_after=None
[2025-09-21T23:54:33.666+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/processamento_silver.py took 0.628 seconds
[2025-09-21T23:55:04.376+0000] {processor.py:161} INFO - Started process (PID=125) to work on /opt/airflow/dags/processamento_silver.py
[2025-09-21T23:55:04.395+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/processamento_silver.py for tasks to queue
[2025-09-21T23:55:04.550+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:55:04.549+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/processamento_silver.py
[2025-09-21T23:55:04.884+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:57:06.046+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/customers_to_silver.sql
[2025-09-21T23:55:04.885+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:55:04.884+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/geolocation_to_silver.sql
[2025-09-21T23:55:04.886+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:55:04.885+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/order_items_to_silver.sql
[2025-09-21T23:55:04.887+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:55:04.886+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/order_payments_to_silver.sql
[2025-09-21T23:55:04.888+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:55:04.887+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/order_reviews_to_silver.sql
[2025-09-21T23:55:04.888+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:55:04.888+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/orders_to_silver.sql
[2025-09-21T23:55:04.889+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:55:04.888+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/products_to_silver.sql
[2025-09-21T23:55:04.896+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:55:04.896+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/sellers_to_silver.sql
[2025-09-21T23:55:04.912+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:55:04.911+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/product_category_name_translation_to_silver.sql
[2025-09-21T23:55:04.913+0000] {processor.py:840} INFO - DAG(s) 'processamento_silver' retrieved from /opt/airflow/dags/processamento_silver.py
[2025-09-21T23:57:56.090+0000] {processor.py:161} INFO - Started process (PID=250) to work on /opt/airflow/dags/processamento_silver.py
[2025-09-21T23:57:56.092+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/processamento_silver.py for tasks to queue
[2025-09-21T23:57:56.094+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:57:56.093+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/processamento_silver.py
[2025-09-21T23:57:56.114+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:57:56.112+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/customers_to_silver.sql
[2025-09-21T23:57:56.115+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:57:56.115+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/geolocation_to_silver.sql
[2025-09-21T23:57:56.116+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:57:56.115+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/order_items_to_silver.sql
[2025-09-21T23:57:56.116+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:57:56.116+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/order_payments_to_silver.sql
[2025-09-21T23:57:56.117+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:57:56.116+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/order_reviews_to_silver.sql
[2025-09-21T23:57:56.117+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:57:56.117+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/orders_to_silver.sql
[2025-09-21T23:57:56.118+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:57:56.118+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/products_to_silver.sql
[2025-09-21T23:57:56.119+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:57:56.119+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/sellers_to_silver.sql
[2025-09-21T23:57:56.120+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:57:56.119+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/product_category_name_translation_to_silver.sql
[2025-09-21T23:57:56.120+0000] {processor.py:840} INFO - DAG(s) 'processamento_silver' retrieved from /opt/airflow/dags/processamento_silver.py
[2025-09-21T23:55:55.101+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:55:55.101+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-09-21T23:55:55.118+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:55:55.118+0000] {dag.py:3823} INFO - Setting next_dagrun for processamento_silver to None, run_after=None
[2025-09-21T23:55:55.146+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/processamento_silver.py took 0.237 seconds
[2025-09-21T23:58:21.276+0000] {processor.py:161} INFO - Started process (PID=305) to work on /opt/airflow/dags/processamento_silver.py
[2025-09-21T23:56:20.115+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/processamento_silver.py for tasks to queue
[2025-09-21T23:56:20.117+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:56:20.116+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/processamento_silver.py
[2025-09-21T23:56:20.344+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:56:20.331+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/customers_to_silver.sql
[2025-09-21T23:56:20.345+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:56:20.344+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/geolocation_to_silver.sql
[2025-09-21T23:56:20.345+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:56:20.345+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/order_items_to_silver.sql
[2025-09-21T23:56:20.346+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:56:20.346+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/order_payments_to_silver.sql
[2025-09-21T23:56:20.347+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:56:20.347+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/order_reviews_to_silver.sql
[2025-09-21T23:56:20.348+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:56:20.347+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/orders_to_silver.sql
[2025-09-21T23:56:20.349+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:56:20.348+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/products_to_silver.sql
[2025-09-21T23:56:20.349+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:56:20.349+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/sellers_to_silver.sql
[2025-09-21T23:56:20.350+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:56:20.350+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/product_category_name_translation_to_silver.sql
[2025-09-21T23:56:20.351+0000] {processor.py:840} INFO - DAG(s) 'processamento_silver' retrieved from /opt/airflow/dags/processamento_silver.py
[2025-09-21T23:56:20.395+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:56:20.394+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-09-21T23:56:20.417+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:56:20.417+0000] {dag.py:3823} INFO - Setting next_dagrun for processamento_silver to None, run_after=None
[2025-09-21T23:56:20.665+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/processamento_silver.py took 0.586 seconds
[2025-09-21T23:58:36.280+0000] {processor.py:161} INFO - Started process (PID=361) to work on /opt/airflow/dags/processamento_silver.py
[2025-09-21T23:58:36.282+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/processamento_silver.py for tasks to queue
[2025-09-21T23:58:36.283+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:58:36.283+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/processamento_silver.py
[2025-09-21T23:56:35.135+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:56:35.132+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/customers_to_silver.sql
[2025-09-21T23:56:35.136+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:56:35.136+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/geolocation_to_silver.sql
[2025-09-21T23:56:35.137+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:56:35.137+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/order_items_to_silver.sql
[2025-09-21T23:56:35.138+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:56:35.138+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/order_payments_to_silver.sql
[2025-09-21T23:56:35.139+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:56:35.138+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/order_reviews_to_silver.sql
[2025-09-21T23:56:35.139+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:56:35.139+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/orders_to_silver.sql
[2025-09-21T23:56:35.140+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:56:35.140+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/products_to_silver.sql
[2025-09-21T23:56:35.141+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:56:35.140+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/sellers_to_silver.sql
[2025-09-21T23:56:35.141+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:56:35.141+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/product_category_name_translation_to_silver.sql
[2025-09-21T23:56:35.142+0000] {processor.py:840} INFO - DAG(s) 'processamento_silver' retrieved from /opt/airflow/dags/processamento_silver.py
[2025-09-21T23:56:35.194+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:56:35.194+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-09-21T23:56:35.228+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:56:35.227+0000] {dag.py:3823} INFO - Setting next_dagrun for processamento_silver to None, run_after=None
[2025-09-21T23:58:36.606+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/processamento_silver.py took 0.334 seconds
[2025-09-21T23:59:46.366+0000] {processor.py:161} INFO - Started process (PID=497) to work on /opt/airflow/dags/processamento_silver.py
[2025-09-21T23:59:46.368+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/processamento_silver.py for tasks to queue
[2025-09-21T23:59:46.369+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:46.369+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/processamento_silver.py
[2025-09-21T23:59:46.390+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:46.388+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/customers_to_silver.sql
[2025-09-21T23:59:46.390+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:46.390+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/geolocation_to_silver.sql
[2025-09-21T23:59:46.391+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:46.391+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/order_items_to_silver.sql
[2025-09-21T23:59:46.391+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:46.391+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/order_payments_to_silver.sql
[2025-09-21T23:59:46.392+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:46.392+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/order_reviews_to_silver.sql
[2025-09-21T23:59:46.393+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:46.392+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/orders_to_silver.sql
[2025-09-21T23:59:46.393+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:46.393+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/products_to_silver.sql
[2025-09-21T23:59:46.394+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:46.394+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/sellers_to_silver.sql
[2025-09-21T23:59:46.394+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:46.394+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/product_category_name_translation_to_silver.sql
[2025-09-21T23:59:46.395+0000] {processor.py:840} INFO - DAG(s) 'processamento_silver' retrieved from /opt/airflow/dags/processamento_silver.py
[2025-09-21T23:57:45.366+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:57:45.365+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-09-21T23:57:45.381+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:57:45.381+0000] {dag.py:3823} INFO - Setting next_dagrun for processamento_silver to None, run_after=None
[2025-09-21T23:57:45.416+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/processamento_silver.py took 0.234 seconds
[2025-09-21T23:59:51.398+0000] {processor.py:161} INFO - Started process (PID=549) to work on /opt/airflow/dags/processamento_silver.py
[2025-09-21T23:59:51.406+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/processamento_silver.py for tasks to queue
[2025-09-21T23:59:51.407+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:51.407+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/processamento_silver.py
[2025-09-21T23:59:51.426+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:51.425+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/customers_to_silver.sql
[2025-09-21T23:59:51.427+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:51.427+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/geolocation_to_silver.sql
[2025-09-21T23:59:51.428+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:51.428+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/order_items_to_silver.sql
[2025-09-21T23:59:51.428+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:51.428+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/order_payments_to_silver.sql
[2025-09-21T23:59:51.429+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:51.429+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/order_reviews_to_silver.sql
[2025-09-21T23:59:51.430+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:51.429+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/orders_to_silver.sql
[2025-09-21T23:59:51.430+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:51.430+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/products_to_silver.sql
[2025-09-21T23:59:51.431+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:51.431+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/sellers_to_silver.sql
[2025-09-21T23:59:51.431+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:51.431+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/product_category_name_translation_to_silver.sql
[2025-09-21T23:59:51.432+0000] {processor.py:840} INFO - DAG(s) 'processamento_silver' retrieved from /opt/airflow/dags/processamento_silver.py
[2025-09-21T23:59:51.450+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:51.450+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-09-21T23:59:51.468+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:51.468+0000] {dag.py:3823} INFO - Setting next_dagrun for processamento_silver to None, run_after=None
[2025-09-21T23:59:51.493+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/processamento_silver.py took 0.102 seconds
[2025-09-21T23:59:11.116+0000] {processor.py:161} INFO - Started process (PID=711) to work on /opt/airflow/dags/processamento_silver.py
[2025-09-21T23:59:11.118+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/processamento_silver.py for tasks to queue
[2025-09-21T23:59:11.119+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:11.119+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/processamento_silver.py
[2025-09-21T23:59:11.139+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:11.137+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/customers_to_silver.sql
[2025-09-21T23:59:11.139+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:11.139+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/geolocation_to_silver.sql
[2025-09-21T23:59:11.140+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:11.140+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/order_items_to_silver.sql
[2025-09-21T23:59:11.141+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:11.141+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/order_payments_to_silver.sql
[2025-09-21T23:59:11.142+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:11.141+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/order_reviews_to_silver.sql
[2025-09-21T23:59:11.143+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:11.142+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/orders_to_silver.sql
[2025-09-21T23:59:11.143+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:11.143+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/products_to_silver.sql
[2025-09-21T23:59:11.144+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:11.144+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/sellers_to_silver.sql
[2025-09-21T23:59:11.145+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:11.144+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/product_category_name_translation_to_silver.sql
[2025-09-21T23:59:11.145+0000] {processor.py:840} INFO - DAG(s) 'processamento_silver' retrieved from /opt/airflow/dags/processamento_silver.py
[2025-09-21T23:59:11.167+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:11.166+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-09-21T23:59:11.196+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:11.196+0000] {dag.py:3823} INFO - Setting next_dagrun for processamento_silver to None, run_after=None
[2025-09-21T23:59:11.249+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/processamento_silver.py took 0.138 seconds
[2025-09-21T23:59:42.209+0000] {processor.py:161} INFO - Started process (PID=764) to work on /opt/airflow/dags/processamento_silver.py
[2025-09-21T23:59:42.211+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/processamento_silver.py for tasks to queue
[2025-09-21T23:59:42.213+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:42.212+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/processamento_silver.py
[2025-09-21T23:59:42.234+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:42.233+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/customers_to_silver.sql
[2025-09-21T23:59:42.235+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:42.234+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/geolocation_to_silver.sql
[2025-09-21T23:59:42.235+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:42.235+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/order_items_to_silver.sql
[2025-09-21T23:59:42.236+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:42.236+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/order_payments_to_silver.sql
[2025-09-21T23:59:42.237+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:42.237+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/order_reviews_to_silver.sql
[2025-09-21T23:59:42.238+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:42.237+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/orders_to_silver.sql
[2025-09-21T23:59:42.238+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:42.238+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/products_to_silver.sql
[2025-09-21T23:59:42.239+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:42.239+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/sellers_to_silver.sql
[2025-09-21T23:59:42.240+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:42.240+0000] {templater.py:95} ERROR - Failed to resolve template field 'sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 93, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 204, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/sql/silver/product_category_name_translation_to_silver.sql
[2025-09-21T23:59:42.240+0000] {processor.py:840} INFO - DAG(s) 'processamento_silver' retrieved from /opt/airflow/dags/processamento_silver.py
[2025-09-21T23:59:42.269+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:42.269+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-09-21T23:59:42.294+0000] {logging_mixin.py:188} INFO - [2025-09-21T23:59:42.288+0000] {dag.py:3823} INFO - Setting next_dagrun for processamento_silver to None, run_after=None
[2025-09-21T23:59:42.329+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/processamento_silver.py took 0.127 seconds
